internal/pkg/scaffold/ansible/k8s_status.go

```diff
@@ -343,8 +343,40 @@ class KubernetesAnsibleStatusModule(KubernetesAnsibleModule):
             'changed': True
         }
 
+    def clean_last_transition_time(self, status):
+        '''clean_last_transition_time removes lastTransitionTime attribute from each status.conditions[*] (from old conditions). 
+        It returns copy of status with updated conditions. Copy of status is returned, because if new conditions 
+        are subset of old conditions, then module would return conditions without lastTransitionTime. Updated status 
+        should be used only for check in object_contains function, not for next updates, because otherwise it can create 
+        a mess with lastTransitionTime attribute.
+
+        If new conditions don't contain lastTransitionTime and they are different from old conditions 
+        (e.g. they have different status), conditions are updated and kubernetes should sets lastTransitionTime 
+        field during update. If new conditions contain lastTransitionTime, then conditions are updated.
+       
+        Parameters:
+          status (dict): dictionary, which contains conditions list
+        
+        Returns:
+          dict: copy of status with updated conditions
+        '''
+        updated_old_status = copy.deepcopy(status)
+        updated_old_condition = []
+
+        for item in status.get('conditions', []):
```

Comment:
use updated_old_status here and remove the deep copy on the next line, item is already a copy that way